DeploymentExperiences.md

```diff
@@ -6,3 +9,22 @@
+`django-admin startproject project_name` initializes Django's settings, database configurations, and Django-specific options.
+`python manage.py startapp app_name` initializes a directory structure that contains the basic files necessary to run an application.
+You can see more about these commands and what files/directories they manage [by reading the documentation here.](https://docs.djangoproject.com/en/2.1/intro/tutorial01/)
+
+**What went well?**
+The documentation provided in the project README.md was extensive, and allowed for a straightforward development process without diving too deep into the concepts. Additional information was provided to help us understand what is happening as we develop the server. The biggest difference in developing in Django is that much of the work happens in a virtual environment. I'm using `pip` and `pipenv shell` for the virtual environment, which manages all of our dependencies locally, rather than system-wide. 
+
+**What challenges did you face?**
+Adjusting to this new way of development took a little time, as I had to ensure I was running the right commands in the right shells. Django comes with a huge assortment of tools and addons that can give impressive functionality out of the box. Configuration is well-documented, but that doesn't mean you won't have to debug a few things. The largest challenges I faced were getting comfortable with virtual environments, and understanding what configurations were necessary for Heroku deployment, specifically using the `.env` and `Procfile` files. Thankfully, Heroku's documentation is extensive, and the Lambda instructors were helpful with any errors I encountered.
+
+**How far did you get?**
+I managed to complete the project MVP by deploying a token-authenticated RESTful API for a simple Notes application that serves Note data from a relational database using JSON. Additionally, this application has an administration panel that allows a superuser to create, edit, and delete users and notes from the database.
+
+**Which docs did you think were the most helpful?**
+The official Django documentation was very useful for a first-timer. They have an extensive documentation that can be as shallow or as deep as you choose. In addition to the docs, there is a large community of coders that contribute to community blog posts and mailing lists that can help further your understanding. Everything is laid out in a way that is accessible, and it is easy to find what you are looking for.
+
+All in all, this project has given me a larger toolkit to consider when choosing to build web applications. Being able to deploy to the web using Python is a useful skill indeed, as Python is one of the most widely used languages [in the world](https://pypl.github.io/PYPL.html).
```

Comment:
Adam, this is great, you have very detailed answered that showcase that you have a strong understanding of Django.